docs/README.md

```diff
@@ -18,6 +18,7 @@
 - [CSS](css)
   - [`styled-components`](css/styled-components.md)
   - [sanitize.css](css/sanitize.md)
+  - Using Sass(css/sass.md)
```

Comment:
That's much better, thank you, but you may need to add the square brackets to make it a link (e.g. `[Using Sass](css/sass.md)`)

See the [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)!